tests/testthat/test-facets.R

```diff
@@ -47,3 +47,49 @@ test_that("3 facets becomes 3 panels", {
   u <- unique(trace.axes.df)
   expect_identical(nrow(u), 3L)
 })
+
+# expect a certain number of _unique_ [x/y] axes
```

Comment:
The file renaming is just to follow some convention: All test files that test for `ggplotly` conversions are named `test-ggplot-*.R`. Notice that other test files (such as `test-unimplemented.R` or `test-plotly-getfigure.R`) test for features other than ggplot --> plotly conversions.